src/main/java/org/mozilla/zest/core/v1/ZestCookie.java

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package org.mozilla.zest.core.v1;
+
+import java.util.Date;
+
+public class ZestCookie {
```

Comment:
Should have a since JavaDoc tag, makes life easier for maintenance and for consumers of the library.
(Same comment for other new methods/constructors/classes.)

Reply:
Okay I do my best =)

...but I dont know if life is easier if an interface `ZestRunner` has the comment "The Interface ZestRunner." ;)

Reply:
Note that I'm not asking you to document it, just add the since tag.